packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -760,8 +760,10 @@ export class MgtPersonCard extends MgtTemplatedComponent {
         parent = parent.parentElement;
       }
 
-      if (parent && (parent as MgtPerson).personDetails) {
-        this.personDetails = (parent as MgtPerson).personDetails;
+      let parentPerson = (parent as MgtPerson).personDetails || parent['personDetailsInternal'];
```

Comment:
Not sure, but could this be types to a MgtPerson instead of accessing the props with the index?

Reply:
@sebastienlevert we created the `personDetailsInternal` as a private property in MgtPerson to hold the loggedin person's details when we had the problem of the person component not working when a user was signed out. Not sure if I should make it a public property and access it as `(parent as MgtPerson).personDetailsInternal`

Reply:
Oh got it. I think we're good with this then! We can keep it that way!